Apps/Src/ReadSwitches.c

```diff
@@ -43,11 +43,13 @@ void Task_ReadSwitches(void* p_arg) {
         if (Switches_Read(IGN_1) == ON || Switches_Read(IGN_2) == ON) {
             Contactors_Enable(ARRAY_CONTACTOR);
             Contactors_Enable(ARRAY_PRECHARGE);
+            Lights_Set(A_CNCTR,ON);
```

Comment:
Don't need the lights set, display is going to be the sole indicator of this.

Reply:
Then again, none of this file really matters since it's pretty much getting nuked later by the Minions rewrite anyways.